Library/Formula/gitlab-ci-multi-runner.rb

```diff
@@ -31,10 +24,12 @@ def install
       # Copy from Makefile
       system "go", "build", "-o", "gitlab-ci-multi-runner", "-ldflags", "-X main.NAME=gitlab-ci-multi-runner -X main.VERSION=#{version} -X main.REVISION=#{commit_sha}"
       bin.install "gitlab-ci-multi-runner"
+      bin.install_symlink "gitlab-ci-multi-runner" => "gitlab-runner"
```

Comment:
`gitlab-ci-multi-runner` doesn’t exist anymore at this point, `bin.install` moved it under `bin/`. Why does it need to be installed under another name?